<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modern Server Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabServer">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <widget class="QFrame" name="sessionFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>221</width>
          <height>37</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="sessionLayout">
         <item>
          <widget class="QLabel" name="labelUptime">
           <property name="styleSheet">
            <string>font-size: 14px;</string>
           </property>
           <property name="text">
            <string>Server Uptime: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="uptimeFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>116</width>
          <height>37</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="uptimeLayout">
         <item>
          <widget class="QLabel" name="labelSession">
           <property name="styleSheet">
            <string>font-size: 14px;</string>
           </property>
           <property name="text">
            <string>Session ID: </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lorem ipsum</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>341</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnStop">
           <property name="text">
            <string>‚èπ Stop Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>‚ñ∂ Start Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSettings">
           <property name="text">
            <string>‚öô Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLogs">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>361</width>
          <height>251</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>359</width>
           <height>249</height>
          </rect>
         </property>
         <widget class="QListView" name="listView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>371</width>
            <height>251</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Lorem ipsum</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnExportLogs">
           <property name="text">
            <string>üì§ Export Logs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEncryptLogs">
           <property name="text">
            <string>üîê Encrypt Logs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteLogs">
           <property name="text">
            <string>üóë Delete Logs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
